build.sh

```diff
@@ -112,7 +112,7 @@ prepare_native_build()
     # Generate version.c if specified, else have an empty one.
     __versionSourceFile=$__scriptpath/bin/obj/version.c
     if [ $__generateversionsource == true ]; then
-        $__scriptpath/Tools/corerun $__scriptpath/Tools/MSBuild.exe "$__scriptpath/build.proj" /t:GenerateVersionSourceFile /p:NativeVersionSourceFile=$__scriptpath/bin/obj/version.c /p:GenerateVersionSourceFile=true /v:minimal
```

Comment:
I can take this approach if you want me to. Given that we are only really running one target in this msbuild invocation, I don't really think there would be any difference as there are hardly any properties that this target relies on. Again, if you feel strongly about it, I can totally move this to a param instead.